### ISSUE TITLE
Ferny 3.1.0 Does not open

### ISSUE THREAD

**Issue by jeremyworger** [ModuleArt/ferny#30]
Hi there, the browser will not open at all after I've installed it. Am I doing it wrong?
Windows 10.

**Comment by jeremyworger** [comment]
Still curious

**Comment by Beelink** [comment]
Does the window just not appear?

**Comment by jeremyworger** [comment]
Yeah, nothing happens at all after installing and clicking to open. However, it is open in my task manager

**Comment by jeremyworger** [comment]
Would it be possible to fix this issue?

**Comment by Beelink** [comment]
Yes, it's possible. I realized that this project is not very promising. It has poor performance compared to Brave or Chrome.
